pages/am/3.x/protocols/uma2/overview.adoc

```diff
@@ -0,0 +1,439 @@
+
+The authorization server and resource server interact with the client and requesting party in a way that is asynchronous with respect to resource owner interactions.
+
+This lets a resource owner configure an authorization server with authorization grant rules (policy conditions) at will, rather than authorizing access token issuance synchronously just after authenticating.
+
+For example, bank customer (resource owner) Alice with a bank account service (resource server) can use a sharing management service (authorization server) hosted by the bank to manage access to her various protected resources by spouse Bob, accounting professional Charline, and financial information aggregation company Decide Account, all using different client applications.
+Each of her bank accounts is a protected resource, and two different scopes of access she can control on them are viewing account data and accessing payment functions.
+
+NOTE: link:https://docs.kantarainitiative.org/uma/wg/oauth-uma-grant-2.0-08.html#claim-redirect[The User-Managed Access (UMA) 2.0 Grant for OAuth 2.0 Authorization specification] discusses the use of the authorization server's claims interaction endpoint for one or more interactive claims-gathering processes as the authorization server requires. Gravitee.io AM does not support interactive claims gathering. Claims gathering is accomplished by having the requesting party acquire an OpenID Connect (OIDC) ID token.
+
+== Using UMA 2.0
+
+Gravitee.io AM exposes an endpoint for discovering information about the UMA provider configuration.
+
+NOTE: Discovery endpoint URL : https://am-gateway/{domain}/uma/.well-known/uma2-configuration
+
+You can also find UMA 2.0 protocol endpoints exposed by Gravitee.io Access Management :
+
+* link:/am/current/uma2/index.html[UMA 2.0 API Console, window="_blank"] - link:/am/current/uma2s/swagger.yml[Swagger spec, window="_blank"]
+
+== Roles
+
+The UMA grant enhances the OAuth definitions of entities in order to accommodate the requesting party role.
+
+* `resource owner`
+
+An entity capable of granting access to a protected resource, the "user" in User-Managed Access.
+The resource owner MAY be an end-user (natural person) or MAY be a non-human entity treated as a person for limited legal purposes (legal person), such as a corporation.
+
+* `requesting party`
+
+A natural or legal person that uses a client to seek access to a protected resource. The requesting party may or may not be the same party as the resource owner.
+
+* `client`
+
+An application that is capable of making requests for protected resources with the resource owner's authorization and on the requesting party's behalf.
+
+* `resource server`
+
+A server that hosts resources on a resource owner's behalf and is capable of accepting and responding to requests for protected resources.
+
+* `authorization server`
+
+A server that protects, on a resource owner's behalf, resources hosted at a resource server.
```

Comment:
```suggestion
resource owner:: An entity capable of granting access to a protected resource, the "user" in User-Managed Access. +
The resource owner MAY be an end-user (natural person) or MAY be a non-human entity treated as a person for limited legal purposes (legal person), such as a corporation.

requesting party:: A natural or legal person that uses a client to seek access to a protected resource. The requesting party may or may not be the same party as the resource owner.

client:: An application that is capable of making requests for protected resources with the resource owner's authorization and on the requesting party's behalf.

resource server:: A server that hosts resources on a resource owner's behalf and is capable of accepting and responding to requests for protected resources.

authorization server:: A server that protects, on a resource owner's behalf, resources hosted at a resource server.
```

Will end up to: 
![image](https://user-images.githubusercontent.com/16895682/86289653-5de22880-bbec-11ea-8427-3e5b54b285a6.png)

Reply:
FP